docs/integrations/keras.rst

```diff
@@ -1,112 +1,319 @@
-.. _integrations-keras:
 
 Neptune-Keras Integration
-=========================
+*************************
 
-|Youtube Video|
+What will you get with this integration
+=======================================
 
-Neptune has implemented an integration with the Keras neural network library.
+In this integration, you'll learn how to efficiently track and monitor model training, log and hardware metrics, images, artifacts and even datasets in any Keras script.
 
-Integration with Keras is enabled through the |neptune-contrib| package. 
-It lets you automatically track metrics and losses (on *batch end* and *epoch end*).
+.. TODO: ADD VIDEO
 
+Video [1min screencast] 
 
-.. image:: ../_static/images/integrations/keras_neptuneml.png
-   :target: ../_static/images/integrations/keras_neptuneml.png
-   :alt: Keras neptune.ai integration
+Introduction
+------------
 
+|Keras| is a high level interactive API that can be used to prototype, build and train neural networks. \
+Keras acts as an interface for the TensorFlow library, and in the latest version of Tensorflow, Keras was officially integrated under the name |tf.keras|.
+
+By the end of this example you will be able to: \
+
+* How to integrate Neptune with Keras to effectively track metrics and model training. 
+* How to easily reproduce experiments and aid collaboration in your project. 
+
+.. Note::
+
+   This integration works with library==tf.keras v0.24.1 and neptune-contrib v0.24.1 
+
+Links:
+|source code| |open in colab|
+
+.. |Keras| raw:: html
+
+    <a href="https://keras.io" target="_blank">Keras</a> 
+
+.. |tf.keras| raw:: html
+
+    <a href="https://www.tensorflow.org/api_docs/python/tf/keras" target="_blank">tf.keras</a>
+
+.. |link| raw:: html
+
+    <a href="https://www.pyimagesearch.com/2019/10/21/keras-vs-tf-keras-whats-the-difference-in-tensorflow-2-0" target="_blank">link</a>
+
+.. |open in colab| raw:: html
+
+    <a href="https://colab.research.google.com/drive/1ITLOePfJh9rNR1jis5gBES2InDo_8Iz4?usp=sharing" target="_blank">open in colab</a>
+
+
+.. |steps here| raw:: html
+
+    <a href="https://docs.neptune.ai/workspace-project-and-user-management/projects/create-project.html" target="_blank">open in colab</a>
+
+
+.. |signup| raw:: html
+
+    <a href="https://neptune.ai/register" target="_blank">open in colab</a>
+
+Quickstart
+==========
+
+**Step 1**
+
+Get a neptune account and your API token. You can |signup| here.
+
+**Step 2**
+
+Create a new project called spam-classifier. You can follow the |steps here| to create a new project. 
```

Comment:
This is not keras-related and should go to the before you start -> please take a look at optuna and lightning integrations

Also, you should log experiments to a public project 'shared/keras-integration` with api_token='ANONYMOUS'

Remember that the doc page is not a blog post and it should be as integration-focused as possible.